test/fixtures/collaboration/apps/web/app/templates/assets/index.html.erb

```diff
@@ -1 +1,2 @@
-<%= image('application.jpg') %>
\ No newline at end of file
+<%= image('application.jpg') %>
+<%= video('movie.mp4') %>
```

Comment:
@leighhalliday Can you please add just another example here, with a video that uses this example:

```erb
<%=
video do
  text "Your browser does not support the video tag"
  source src: view.asset_path('movie.mp4'), type: 'video/mp4'
  source src: view.asset_path('movie.ogg'), type: 'video/ogg'
end
%>
```

So we are sure that Lotus::View is able to render the expected markup. Thank you!